### PR TITLE
PMM-9682 large scale support

### PR DESCRIPTION
**Pull request by BupycHuk**
PMM-9682

Build: SUBMODULES-2689

- [ ] Links to other linked pull requests (optional).

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -165,6 +164,8 @@ eks-delete-current-namespace:
...
 	if [ "$$NAMESPACE" != "default" ]; then kubectl delete ns "$$NAMESPACE"; fi
 
-deploy-to-pmm-server: install     ## Deploy DBaaS controller to a running ${PMM_CONTAINER} container.
-	docker cp bin/dbaas-controller ${PMM_CONTAINER}:/usr/sbin/dbaas-controller
-	docker exec ${PMM_CONTAINER} supervisorctl restart dbaas-controller
+deploy-to-pmm-server: release     ## Deploy DBaaS controller to a running ${PMM_CONTAINER} container.
+	docker exec ${PMM_CONTAINER} supervisorctl stop dbaas-controller
+	docker cp $(PMM_RELEASE_PATH)/dbaas-controller ${PMM_CONTAINER}:/usr/sbin/dbaas-controller
+	docker exec ${PMM_CONTAINER} supervisorctl start dbaas-controller
+	docker exec ${PMM_CONTAINER} supervisorctl tail -f dbaas-controller
```
Comment (gen1us2k) [inline Makefile:171]: 👍

**Review by gen1us2k — COMMENTED** [review]
Looks good to me. The only thing that concerns me is lowered coverage.

**Comment by BupycHuk** [comment]
@gen1us2k good point, will add tests for new endpoints.

**Review by percona-csalguero — APPROVED** [review]

**Review by gen1us2k — APPROVED** [review]

**Comment by gen1us2k** [comment]
We moved to dbaas-operator and I'm closing this. Thank you for your contribution

**Pull request closed without merging**